TACC/Core-Portal
# bug/FP-1211: Manage Account Section Errors Act like Giant Panels

## Overview: ##
Reduced both size and number of manage account errors.

## Related Jira tickets: ##

* [FP-1211](https://jira.tacc.utexas.edu/browse/FP-1211)

## Summary of Changes: ##
- Added a new style class for the error message which overrides `.panels` 
- Wrapped error message in div with above style class
- Altered panels height to accommodate error panel

## Testing Steps: ##
In server/portal/apps/accounts/views.py:
1. At line 104, add: `l['test'] = l.test`
2. Confirm that only the '... Information' tables and 'Change Password' panel render along with the error message, as in the UI photo below.


## UI Photos:
![image](https://user-images.githubusercontent.com/43251554/140971050-f0f60c29-30ef-443a-8caa-90f00c88e687.png)

## Notes: ##
This testing step was found accidentally when working on another ticket, and therefore is not very intentional in testing this change. If there are other suggestions for generating errors here, they would be appreciated.
